storage/rocksdb/ha_rocksdb.h

```diff
@@ -1248,4 +1251,24 @@ extern std::atomic<uint64_t> rocksdb_binlog_ttl_compaction_timestamp;
 extern bool rocksdb_enable_tmp_table;
 extern bool rocksdb_enable_delete_range_for_drop_index;
 extern bool rocksdb_disable_instant_ddl;
+
+extern char *rocksdb_wal_dir;
+extern char *rocksdb_datadir;
+
+extern uint rocksdb_clone_checkpoint_max_age;
+extern uint rocksdb_clone_checkpoint_max_count;
+
+// TODO(laurynas): is this the right place for these functions?
```

Comment:
This is fine and you can remove this TODO.

Reply:
Done